frontend/src/components/Item/ItemMenu.vue

```diff
@@ -33,7 +33,7 @@
       </v-list>
     </v-menu>
     <metadata-editor-dialog
-      v-if="metadataDialog"
+      v-if="item.Id"
```

Comment:
`v-if` evaluations won't get added to the DOM (at least to my knowledge and I quickly checked the DOM).
This `v-if` directive is here because the MetadataEditorDialog component always expects a defined `item-id` parameter, and because item.Id could be null or undefined, adding the `v-if` narrows the type to remove null or undefined and satisfies the type checking. In practice it will always be true and is only there to satisfy typings.

Reply:
Also, almost nothing is added to the DOM by just adding metadata-editor-dialog, as it's a v-dialog the content is also v-ifed by Vuetify

Reply:
@ThibaultNocchi @aweebs This was not the case with Vuetify 2. Are we sure that in Vuetify 3 everything is v-ified? Can any of you check please? I'm afk right now.

In case everything is v-ified now, forget about this.

Reply:
Confirmed that nothing in the component is mounted into the DOM when the dialog is not opened.